templates/taxi/car_list.html

```diff
@@ -2,16 +2,19 @@
 
 {% block content %}
   <h1>Car list</h1>
-  {% if car_list %}
+  
+  <a href="{% url 'taxi:car-create' %}" class="btn btn-primary mb-2">Create New Car</a>
```

Comment:
Ensure that the user has the necessary permissions to create a new car. Consider adding permission checks if they are not already implemented.